config/oval.config

```diff
@@ -31,4 +31,4 @@ multi_platform_rhev =
 # OVAL Schema version
 #
 [Schema Version]
-oval_version = 5.10
+oval_version = 5.11
```

Comment:
Is there a supported `openscap-scanner` that works with OVAL 5.11 on RHEL 6 & 7?